src/kernels/composable_kernel/include/kernel_algorithm/gridwise_convolution_implicit_gemm_v4r4_gen_xdlops_gnchw_gkcyx_gnkhw_lds_double_buffer.hpp

```diff
@@ -265,7 +265,8 @@ struct GridwiseConvolutionImplicitGemm_v4r4_gen_xdlops_gnchw_gkcyx_gnkhw_lds_dou
             2,
             GemmBBlockCopySrcDataPerRead_GemmN,
             GemmBBlockCopyDstDataPerWrite_GemmN,
-            CGlobalMemoryDataOperation>{};
+            CGlobalMemoryDataOperation,
+            ConvStrideW>{};
```

Comment:
why there is stride specified for B but not for A?

Reply:
Stride always happens in B matrix, right?

Reply:
Fixed.